src/test/java/za/ac/cput/repository/LecturerRepositoryImplTest.java

```diff
@@ -9,8 +9,13 @@
 import static org.junit.jupiter.api.Assertions.*;
 @TestMethodOrder(MethodOrderer.MethodName.class)
 class LecturerRepositoryImplTest {
+<<<<<<< HEAD
+    private static LecturerRepositoryImpl repository = LecturerRepositoryImpl.createLectureRepository();
+    private static Lecturer lecturer = LecturerFactory.createLecturer(1, "Arinze", "Anikwue");
+=======
     private static final LecturerRepositoryImpl repository = LecturerRepositoryImpl.createLectureRepository();
     private static final Lecturer lecturer = LecturerFactory.createLecturer("Arinze", "Anikwue");
+>>>>>>> cc76966ecbbc0f8c68525a302105f231ee648e71
 
     @Test
     void a_create() {
```

Comment:
Please name your methods using the camelCasing naming conventions for methods, not Snake_Casing

It should be `void create()` **not** `void a_create()`
For consistency within the project, please re-name it `save()` also `getAll()` should be `findAll()` 

Please do so with the remaining methods.